Rakefile

```diff
@@ -68,3 +70,13 @@ If you know some other tricks, please contribute!
 }
   end
 end
+
+desc 'Adds the given name a contributor'
+task :add_contributor do |t, args|
+  contributor = ARGV[1]
+  raise ArgumentError, 'Contributor name is required' if contributor.nil?
+  Contributor.new(contributor).add
+  Rake::Task['build'].execute
+  puts "#{contributor} was added as a contributor"
+  exit
```

Comment:
rake usa os parâmetros como diferentes rake tasks e chamaria uma task inexistente com o nome do colaborador se não forçamos a saída do bloco @filipebarcos.